netsocadmin/routes/tools.py

```diff
@@ -183,13 +188,16 @@ def dispatch_request(self) -> str:
         # Check that all fields are valid
         valid, msg = self.validate(username, password, dbname)
         if not valid:
+            self.logger.debug(f"error {msg}")
             return self.render(mysql_error=msg, mysql_active=True)
         # Try to create the database
         try:
             mysql.create_database(username, dbname, False)
         except mysql.DatabaseAccessError as e:
+            self.logger.debug(f"database error {e.__cause__}")
             return self.render(mysql_error=e.__cause__, mysql_active=True)
         # Success (probably should do more than just redirect to / ...)
+        self.logger.debug("successful")
```

Comment:
The log points here need some context, if reading from the log we'll have little info as to where this was called from